jitbuilder/lljb/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+###############################################################################
+# Copyright (c) 2019, 2019 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at http://eclipse.org/legal/epl-2.0
+# or the Apache License, Version 2.0 which accompanies this distribution
+# and is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception [1] and GNU General Public
+# License, version 2 with the OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+#############################################################################
+
+if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+    set(ENV{PATH} "/usr/local/opt/llvm\@8/bin/:${PATH}")
+endif()
```

Comment:
Is this meant to serve as a guess for find_package(LLVM) below? If so, would it be better to just hand this path to find_package, using the PATHS option?

Reply:
Yup, I have changed this to provide this path to `find_package` directly. It was not clear to me in the documentation, but if I do specify a `PATHS` value, are the paths in $PATH also searched as a fallback? If that's not the case, if llvm version 8 is the default and the only installation a macOS environment, CMake will not find LLVMConfig.cmake.